### PR TITLE
make some small improvements on the technical features page

### PR DESCRIPTION
**Pull request by diegomrsantos**
### Description

Just some small improvements on the technical features page.

### Checklist

- [ ] I have added a label to this PR 🏷️
- [ ] I have run my changes through Grammarly
- [ ] If this page requires a disclaimer, I have added one
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects

### CODE REVIEW

**Review by augustocollado — CHANGES_REQUESTED** [review]

File: learn/tanssi/technical-features.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ In this article, the necessary aspects to consider when building and deploying y
 
 Polkadot is a heterogeneous multi-chain ecosystem, where multiple parallel blockchains connect to a central blockchain called the relay chain. To provide block production as a service, the Tanssi protocol masterfully orchestrates and designs the interaction between the Tanssi network, the appchains, and Polkadot's relay chain.
 
-In the Polkadot ecosystem, the Tanssi appchains connected to the relay chain are fully sovereign blockchains, having their own rules, consensus mechanisms, and so forth, and this is the case for the Tanssi network and the appchains deployed through Tanssi as well.
+In the Polkadot ecosystem, the parachains connected to the relay chain are fully sovereign blockchains, having their own rules, consensus mechanisms, and so forth, and this is the case for the Tanssi network and the appchains deployed through Tanssi as well.
```
[the remark this answers is not in the document]
Comment (diegomrsantos) [reply learn/tanssi/technical-features.md:25]: Are Tanssi appchains and appchains deployed through Tanssi different things?

[the remark this answers is not in the document]
Comment (augustocollado) [reply learn/tanssi/technical-features.md:25]: No, they are synonyms

File: learn/tanssi/technical-features.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Among many other responsibilities, the relay chain validates and finalizes the b
 
 ![Relay chain](/images/learn/tanssi/technical/technical-4.webp)
 
-As mentioned, the Tanssi network and the appchains are sibling chains with no hierarchical dependency. They are communicated via the relay chain, and therefore, the relay chain plays a key role in the protocol.
+As mentioned, the Tanssi network and the appchains are sibling chains with no hierarchical dependency. They communicate via the relay chain, and therefore, the relay chain plays a key role in the protocol.
```
Comment (albertov19) [inline learn/tanssi/technical-features.md:73]: Thanks - @augustocollado please include this typo in one of the PRs we have open

Comment (augustocollado) [reply learn/tanssi/technical-features.md:73]: done!

**Comment by eshaben** [comment]
Thanks for opening the PR @diegomrsantos! 💜 

Closing this PR, as we've applied your suggestion in a separate PR.

**Pull request closed without merging**